src-java/floodlightrouter-topology/floodlightrouter-storm-topology/src/main/java/org/openkilda/wfm/topology/floodlightrouter/FloodlightRouterTopology.java

```diff
@@ -134,7 +168,8 @@ private void speakerToFlowHs(TopologyBuilder topology, TopologyOutput output) {
 
     private void flowHsToSpeaker(TopologyBuilder topology, TopologyOutput output) {
         declareKafkaSpoutForAbstractMessage(topology,
-                kafkaTopics.getSpeakerFlowHsTopic(), ComponentType.SPEAKER_FLOW_HS_KAFKA_SPOUT);
+                kafkaTopics.getSpeakerFlowHsTopic(), ComponentType.SPEAKER_FLOW_HS_KAFKA_SPOUT,
+                getZkTopoName(), getConfig().getBlueGreenMode());
```

Comment:
Does it make sense to introduce a dedicated "build" method instead of repeating "getZkTopoName(), getConfig().getBlueGreenMode()" each time?

Reply:
the original idea was to create a follow up patch to update this declare methods once all components start using zero-downtime spouts/bolts